ee/allowedcmd/cmd_linux.go

```diff
@@ -88,7 +88,7 @@ func Lsof(ctx context.Context, arg ...string) (*exec.Cmd, error) {
 }
 
 func NixEnv(ctx context.Context, arg ...string) (*exec.Cmd, error) {
-	return validatedCommand(ctx, "/nix/var/nix/profiles/default/bin/nix-env", arg...)
+	return validatedCommand(ctx, "/run/current-system/sw/bin/nix-env", arg...)
```

Comment:
Is there any chance we'd need both of these? Because we could put both into the command lists